### PR TITLE
Update display NFTs, pages UI and prettier applied

### PR DESCRIPTION
**Pull request by RichardIrala**
# Home

image

image

image

image

# Inventory

image

image

image

image

image

### CODE REVIEW

**Comment by RichardIrala** [comment]
Only the UI was affected in this pull request

**Comment by RichardIrala** [comment]
Files changed:
* All atoms
* All pages
* All styles files

File: atoms/Item/Item.jsx [hunk 1]
```diff
@@ -35,19 +35,36 @@ const Item = ({ nft, XerialWalletViewmodel }) => {
...
-      <button className="atom-item__button">Buy</button>
+      <h1 className="atom-item__generalText global-style__textWithDots">Token typeId: {nft.typeId}</h1>
+      <h2 className="atom-item__generalText global-style__textWithDots">Token name: {nft.metadata.name}</h2>
+      <p className="atom-item__generalText">Collection address: {nft.metadata.contract.address}</p>
+      <p className="atom-item__generalText">Collection name: {nft.metadata.contract.name}</p>
+      <p className="atom-item__generalText">Seller address: {sellerAddress}</p>
+      <p className="atom-item__generalText">Owner: {sellerAddress}</p>
+      <p className="atom-item__generalText">Original supply: {nft.supply}</p>
+      <p className="atom-item__generalText">Available supply: {nft.supply - nft.mintedTokens}</p>
+      <p className="atom-item__generalText">Minted nfts: {nft.mintedTokens}</p>
```
Comment (luislucena16) [inline atoms/Item/Item.jsx]: rename `Minted nfts` to `Minted NFTs`

File: atoms/SecondaryMarketItem/SecondaryMarketItem.jsx [hunk 2]
```diff
@@ -17,12 +17,24 @@ const SecondaryMarketItem = ({ nft, XerialWalletViewmodel }) => {
...
     >
-      <h1 className="global-style__textWithDots">Token typeId: {nft.typeId}</h1>
-      <h2 className="global-style__textWithDots">Token name: {nft.metadata.name}</h2>
       <img className="atom-item__itemImage" src={nft.metadata.image} alt="nft img" />
-      <p className="atom-item__generalText">collectionAddress: {nft.metadata.contract.address}</p>
-      <p className="atom-item__generalText">supply: 1</p>
-      <p className="atom-item__generalText">price: {nft.price}</p>
+      <h1 className="atom-item__generalText global-style__textWithDots">Token Id: {nft.tokenId}</h1>
+      <h2 className="atom-item__generalText global-style__textWithDots">Token name: {nft.metadata.name}</h2>
+      <p className="atom-item__generalText">CollectionAddress: {nft.metadata.contract.address}</p>
```
Comment (luislucena16) [inline atoms/SecondaryMarketItem/SecondaryMarketItem.jsx]: use this name `Collection address`

File: atoms/SecondaryMarketItem/SecondaryMarketItem.jsx [hunk 3]
```diff
@@ -17,12 +17,24 @@ const SecondaryMarketItem = ({ nft, XerialWalletViewmodel }) => {
...
+      <p className="atom-item__generalText">CollectionAddress: {nft.metadata.contract.name}</p>
```
Comment (luislucena16) [inline atoms/SecondaryMarketItem/SecondaryMarketItem.jsx]: use this name `Collection address`

File: pages/inventory.js [hunk 4]
```diff
@@ -154,24 +168,46 @@ const Inventory = () => {
...
+            <p className="inventory__project-session">SessionToken: {sessionToken}</p>
+          </div>
+          <div className="inventory__items-container">
+            <Items />
+            <SecondaryMarketItems />
+          </div>
+        </div>
+      ) : (
+        <div className="inventory__noLoguedMessageContainer">
+          <p className="inventory__project-session">You are not logged in. Please log in</p>
```
Comment (luislucena16) [inline pages/inventory.js:208]: put a dot (`.`) at the end of this `Please log in.`

File: pages/inventory.js [hunk 5]
```diff
@@ -154,24 +168,46 @@ const Inventory = () => {
...
       </div>
-      <p>User address: {userAddress}</p>
-      <p>SessionToken: {sessionToken}</p>
-      <Items />
-      <SecondaryMarketItems />
+      {sessionToken ? (
+        <div className="inventory__background">
+          <div className="inventory__project-container">
+            <p className="inventory__project-session">User address: {userAddress}</p>
+            <p className="inventory__project-session">SessionToken: {sessionToken}</p>
```
Comment (luislucena16) [inline pages/inventory.js]: rename this `SessionToken:` to `Session token:`

File: atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx [hunk 6]
```diff
@@ -6,11 +6,15 @@ const InventoryItemOnSecondaryMarket = ({ nft }) => {
...
   return (
-    <div style={{ backgroundColor: "rgb(59, 109, 152)", padding: 10, display: "flex", flexDirection: "column", gap: 12, width: 600 }}>
-      <h1>Nft typeId: {nft.typeId}</h1>
-      <h2>Nft name: {nft.metadata.name}</h2>
-      <img src={nft.metadata.image} alt={nft.metadata.name} style={{ width: 300, height: "auto" }} />
-      <p>collectionAddress: {nft.metadata.contract.address}</p>
+    <div className="inventory-items__container">
+      <div className="inventory-items__projectContainer">
+        <img src={nft.metadata.image} alt={nft.metadata.name} className="inventory-items__image" />
+        <h1>Nft marketItemId: {nft.marketItemId}</h1>
```
Comment (luislucena16) [inline atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx]: use this: `NFT market itemId`

File: atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx [hunk 7]
```diff
@@ -6,11 +6,15 @@ const InventoryItemOnSecondaryMarket = ({ nft }) => {
...
+        <h2>Nft name: {nft.metadata.name}</h2>
```
Comment (luislucena16) [inline atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx]: rename to `NFT name`

File: atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx [hunk 8]
```diff
@@ -6,11 +6,15 @@ const InventoryItemOnSecondaryMarket = ({ nft }) => {
...
+        <p>collectionAddress: {nft.metadata.contract.address}</p>
```
Comment (luislucena16) [inline atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx]: rename to: `Collection address`

File: atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx [hunk 9]
```diff
@@ -6,11 +6,15 @@ const InventoryItemOnSecondaryMarket = ({ nft }) => {
...
+        <p>collectionName: {nft.metadata.contract.name}</p>
```
Comment (luislucena16) [inline atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx]: rename to: `Collection name`

File: atoms/SecondaryMarketItem/SecondaryMarketItem.jsx (hunk 2 above)
Comment (RichardIrala) [reply atoms/SecondaryMarketItem/SecondaryMarketItem.jsx]: Done

File: pages/inventory.js (hunk 4 above)
Comment (RichardIrala) [reply pages/inventory.js:208]: Done

File: atoms/Item/Item.jsx (hunk 1 above)
Comment (RichardIrala) [reply atoms/Item/Item.jsx]: Done

File: atoms/SecondaryMarketItem/SecondaryMarketItem.jsx (hunk 3 above)
Comment (RichardIrala) [reply atoms/SecondaryMarketItem/SecondaryMarketItem.jsx]: Done (Replaced with Collection name)

File: atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx (hunk 6 above)
Comment (RichardIrala) [reply atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx]: Done

File: pages/inventory.js (hunk 5 above)
Comment (RichardIrala) [reply pages/inventory.js]: Done

File: atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx (hunk 7 above)
Comment (RichardIrala) [reply atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx]: Done

File: atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx (hunk 8 above)
Comment (RichardIrala) [reply atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx]: Done

File: atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx (hunk 9 above)
Comment (RichardIrala) [reply atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx]: Done

**Review by luislucena16 — APPROVED** [review]

**Comment by luislucena16** [comment]
approve!

**Pull request merged**